package.json

```diff
@@ -14,6 +15,7 @@
     "pngjs": "^3.3.3"
   },
   "devDependencies": {
+    "@types/node": "^10.12.3",
```

Comment:
Why we should add this dependency? Moreover if you add new dependency to `package.json` you should add it to `package.lock.json` too.

Reply:
Updated `package.lock.json`